apps/mobile/src/queries/balance/total-balance.query.ts

```diff
@@ -0,0 +1,52 @@
+import { useGetStacksAddresses } from '@/features/balances/stacks/use-get-stacks-addresses';
+import { AccountId } from '@/models/domain.model';
+import {
+  useBitcoinAccountTotalBitcoinBalance,
+  useWalletTotalBitcoinBalance,
+} from '@/queries/balance/bitcoin-balance.query';
+import { useStxBalance } from '@/queries/balance/stacks-balance.query';
+
+import { Money } from '@leather.io/models';
+import { sumMoney } from '@leather.io/utils';
+
+interface TotalBalance {
+  btcBalance: Money;
+  btcBalanceUsd: Money;
+  stxBalance: Money;
+  stxBalanceUsd: Money;
+  totalBalance: Money;
+}
```

Comment:
Maybe we should use naming `balanceFiat` to more generically apply to users preference

Reply:
I was explicitly calling it USD as `i18nFormatCurrency ` [is hardcoded to only accept USD](https://github.com/leather-io/mono/pull/590#discussion_r1827469125). 

I agree it's bad naming here though so maybe I should change it to `balanceFiat` now and create a task / project for adding multiple currency support?